### PR TITLE
FreemarkerTemplateProcessor: Use LicenseView constants directly

### PR DESCRIPTION
**Pull request by MarcelBochtler**
Instead of searching for it by name.

### CODE REVIEW

File: reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt [hunk 1]
```diff
@@ -120,6 +118,7 @@ class FreemarkerTemplateProcessor(
             "packages" to packages,
             "ortResult" to input.ortResult,
             "licenseTextProvider" to input.licenseTextProvider,
+            "LicenseView" to LicenseView,
```
Comment (sschuberth) [inline reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt:121]: TBH I'm (positively) surprised that it works to not pass an instance here, but a class. And not even just the companion object of the class. But I assume you've verified that this indeed works as expected?

Comment (sschuberth) [reply reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt:121]: It actually does *not* seem to work: https://dev.azure.com/oss-review-toolkit/ort/_build/results?buildId=4068&view=logs&j=0b98c9ad-b4dd-509f-97af-ff0d5e511683&t=d4a109d9-63b6-59b7-7349-067146a0a444&l=3600

Comment (sschuberth) [reply reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt:121]: Maybe there's also a misunderstanding, when looking at the commit message:

> Use LicenseView enum directly

`LicenseView` is not an `enum`, and `CONCLUDED_OR_DECLARED_AND_DETECTED` etc. are not `enum` members, but those are constants in a companion object.

Comment (MarcelBochtler) [reply reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt:121]: > It actually does not seem to work

It does work. I just forgot change some test templates.

**Comment by sschuberth** [comment]
Interesting, these changes seem to have reduced the PDF size quite a bit:

https://dev.azure.com/oss-review-toolkit/ort/_build/results?buildId=4069&view=logs&j=0b98c9ad-b4dd-509f-97af-ff0d5e511683&t=d4a109d9-63b6-59b7-7349-067146a0a444&l=3309

Can you please double-check that this is not due to missing content from the PDF?

**Comment by sschuberth** [comment]
> Can you please double-check that this is not due to missing content from the PDF?

I did a quick check myself, for some reason "MIT WITH Libtool-exception" is now missing from the "License Texts" appendix.

**Comment by MarcelBochtler** [comment]
> Can you please double-check that this is not due to missing content from the PDF?

Fixed it. Had some changed from debugging in there. Sorry for that.

**Review by sschuberth — APPROVED** [review]

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**